demos/security-trifecta/android/app/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
             ignoreAssetsPattern '!.svn:!.git:!.ds_store:!*.scc:.*:!CVS:!thumbs.db:!picasa.ini:!*~'
         }
         manifestPlaceholders = [
-            'AUTH_URL_SCHEME': 'io.ionic.acdemo'
+            'AUTH_URL_SCHEME': 'msauth'
```

Comment:
Are we able to use a scheme that isn't `msauth`? Historically, I've seen customers assume this means they have to use Azure.

Reply:
This depends on the backend being used. For our Azure instance we must use `msauth` for our other backends I have set up other URL schemes.

Reply:
I think that changed because I brought over my changes from the previous repo I used to work out of for this. As to why I had it set to `msauth`, I don't recall. 

Anyway, I agree and its changed back to `io.ionic.demo` now